ext/ffi/php_ffi.h

```diff
@@ -19,6 +19,9 @@
 #ifndef PHP_FFI_H
 #define PHP_FFI_H
 
+#include "php_version.h"
+#define ZEND_FFI_VERSION PHP_VERSION
```

Comment:
Why is this moved to the header?

Reply:
I don't know actually. Because all other extensions mostly have extension version defined in the header. Should this be defined in the *.c file?

Reply:
Basically: Unless there is a good reason for something to be in an exported header, it should always be in the C file. You can actually drop the `ZEND_FFI_VERSION` constant entirely and directly use `PHP_VERSION` in the module info.

Reply:
Changed... Yes, it works directly also fine actually.